data/templated/example.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<query_templates>
+   <query_template>
+      <name>GetOrder</name>
+      <query><![CDATA[SELECT NO_O_ID FROM new_order WHERE NO_D_ID = ? OR NO_D_ID = ? ORDER BY NO_O_ID ASC]]></query>
+      <parameter_types>4,4</parameter_types>
```

Comment:
```suggestion
      <parameter_types>
          <parameter_type>Integer</parameter_type>
          <parameter_type>Integer</parameter_type>
      </parameter_types>
```
This format might also make more sense.

Reply:
And then something like this for `<parameter_values>`:

```xml
    <query_values>
        <parameter_values>
            <parameter_value>3</parameter_value>
            <parameter_value>4</parameter_value>
        </parameter_values>
        <parameter_values>
            <parameter_value>5</parameter_value>
            <parameter_value>6</parameter_value>
        </parameter_values>
    </query_values>
```
More verbose, but also more natural XML object parsing.